lib/middlewares/auth.js

```diff
@@ -21,9 +22,32 @@ passport.use(
     },
     async (req, dni, contrasenia, done) => {
       try {
-        const data = await Usuario.create(req.body);
+        let data = await Usuario.create(req.body);
         const token = generarJWT(data.id, dni);
+        const carreras = await inscripcionesPara(data.dni);
+
+        if (carreras != null) {
+          try {
+            carreras.forEach(async (carrera) => {
+              await InscripcionCarrera.create({
+                usuarioId: data.id,
+                idCarrera: carrera.id,
+                nombreCarrera: carrera.nombre,
+              });
+            });
```

Comment:
Ojo, que esto no espera a que las promises terminen. Te sugiero hacerlo así:

```suggestion

            await InscripcionCarrera.bulkCreate(
              carreras.map(({id, nombre}) => ({ usuarioId: data.id, idCarrera: id, nombreCarrera: nombre })          
            );
```